src/booleans.js

```diff
@@ -1,59 +1,152 @@
 export const negate = (a) => {
-  // your code here
+    if (a == true) {
```

Comment:
Always use strictly equals `===`

Reply:
I think perhaps you've used `==` here because you want to check for a truthy value, not strictly `true`. You can achieve this with just `if (a) { ... }`